app/mailers/user_mailer.rb

```diff
@@ -6,6 +6,26 @@ class UserMailer < Devise::Mailer
 
   helper_method :suspension_time, :account_name, :instance_name, :locked_time, :unlock_time
 
+  def two_step_reset(user)
+    @user = user
+    mail(to: @user.email, subject: "2-step verification is now reset")
+  end
+
+  def two_step_changed(user)
+    @user = user
+    mail(to: @user.email, subject: "2-step verification phone changed successfully")
+  end
+
+  def two_step_enabled(user)
+    @user = user
+    mail(to: @user.email, subject: "2-step verification set up")
+  end
+
+  def two_step_flagged(user)
+    @user = user
+    mail(to: @user.email, subject: "Set up 2-step verification")
```

Comment:
I should have mentioned, like the body, the subjects are WIP placeholders for now. I'd assume the content team provides both subject and body? Happy to go with your suggestions though if they're definitive. :+1: